lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -1035,6 +1035,11 @@ def install_repo_configs_from_url(host, repo_config_url, copy_dir = nil)
           end
           scp_to( host, repo, to_path )
 
+          variant, version, arch, codename = host['platform'].to_array
+          if variant =~ /^ubuntu$/ && version.split('.').first.to_i >= 18
+            # Allow the use of unsigned repos with Ubuntu 18.04+
+            on host, "echo 'Acquire::AllowInsecureRepositories \"true\";' > /etc/apt/apt     .conf.d/90insecure"
```

Comment:
Ah, I should have noticed this before running the tests, but the spaces in the filename are stopping this from working -- this line is applied in the correct tests though.